client/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I don't know if this can be removed (or what it is used for, but I guess we can change the content to be about "verejne digital")